kDriveCore/Data/Cache/AccountManager.swift

```diff
@@ -434,4 +451,22 @@ public class AccountManager: RefreshTokenDelegate, AccountManageable {
     public func account(for userId: Int) -> Account? {
         return accounts.first { $0.userId == userId }
     }
+
+    public func logoutCurrentAccountAndSwitchToNextIfPossible() {
+        Task { @MainActor in
+            if let currentAccount {
+                removeTokenAndAccount(account: currentAccount)
+                notificationHelper.sendDisconnectedNotification()
```

Comment:
This displays a snackbar or a notification when the user disconnect manually (from the menu) but this wasn't the case before

Reply:
We displayed a snackbar when logged out automatically. I figured I could put it in the factorised method.
I got you, we only use the snackbar when the logout is automatic. Yeah we have enough snackbar already.